edgelet/edgelet-http/src/authentication.rs

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft. All rights reserved.
 
-#![deny(rust_2018_idioms, warnings)]
+#![deny(warnings)]
```

Comment:
Might as well remove these entirely, just like https://github.com/Azure/iotedge/commit/2a394e57dcbca7bd4c4c6d28aaccbc6d8ef5662f did in master.